dependency-check-maven-config.xml

```diff
@@ -117,24 +117,6 @@
         <cve>CVE-2017-12634</cve>
     </suppress>
 
```

Comment:
❗️We also need to remove https://github.com/stustison/ddf/blob/2574cafdfc176657e434d01b80a5bb06763cd6eb/dependency-check-maven-config.xml#L3-L7.